app/js/userScores.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * This function gets the user data and grabs all the percentage scores
+ *
+ * @returns will return an array of strings of percentage scores and passes it as a param in within a function call for categoriseData
+ */
+async function userScores(){
+    let usersObject = await createUsersObject()
+    let usersData = usersObject.data
+    let percentagesArray = []
+    usersData.forEach(function (element) {
+        percentagesArray.push(element.percentage)
+    })
+    categoriseData(percentagesArray)
+}
+
+/**
+ * This function divides user scores into three categories
+ *
+ * @param is the user percentage scores array generated by the userScores function
+ *
+ * @returns will return a single array giving the total number of users falling into each score category
+ */
+
+function categoriseData(userScores){
+    let highscore = []
+    let pass = []
+    let notPass = []
+    userScores.forEach(function (element) {
+        if (parseInt(element) >= 97){
+            highscore.push(element)
+        } else if (parseInt(element) >= 70 && parseInt(element) < 97) {
+            pass.push(element)
+        } else {
+            notPass.push(element)
+        }
+    })
+    let highscoreTotal = highscore.length
+    let passTotal = pass.length
+    let notPassTotal = notPass.length
+    let categorisedScores = [highscoreTotal, passTotal, notPassTotal]
+    return categorisedScores
+}
```

Comment:
Could be written like this to shorten function

```suggestion
}function categoriseData(userScores){
    let highscore = 0
    let pass = 0
    let notPass = 0
    userScores.forEach(function (element) {
        if (parseInt(element) >= 97){
            highscore += 1
        } else if (parseInt(element) >= 70 && parseInt(element) < 97) {
            pass += 1
        } else {
            notPass += 1
        }
    })
    let categorisedScores = [highscore, pass, notPass]
    console.log(categorisedScores)
    return categorisedScores
}
```

Reply:
Change done